modules/commerce_bluesnap_currency/commerce_bluesnap_currency.services.yml

```diff
@@ -0,0 +1,10 @@
+services:
+  commerce_bluesnap_currency.exchange_rate_bluesnap:
+    class: Drupal\commerce_bluesnap_currency\EventSubscriber\ExchangeRateBluesnap
+    tags:
+      - { name: event_subscriber }
+    arguments: ['@config.factory', '@logger.factory']
```

Comment:
**CHANGE REQUEST**

We do have a logger for the module: https://github.com/shabananavas/drupal8-commerce-bluesnap/blob/401f71366d0fb1f9cfcd14d67cb39414c0e10073/commerce_bluesnap.services.yml#L2

Reply:
Done.